client/src/components/common/AclBaseArticleResult.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { BaseAclArticle } from '../../shared/Models';
+import { Heading3, Link } from '../../shared/Styles';
+
+interface BaseArticleResultProps {
+  article: BaseAclArticle;
+  position?: number;
+  onClickTitle?: () => void;
+  boldTitle?: boolean;
+}
+
+const AclBaseArticleResult: React.FC<BaseArticleResultProps> = ({
+  article,
+  position,
+  onClickTitle = () => {},
+  boldTitle = false,
+}) => {
+  let authorString = '';
+  if (article.authors.length > 0) {
+    article.authors.forEach((author, idx) => {
+      if (author !== '') {
+        authorString += idx === article.authors.length - 1 ? `${author}.` : `${author}, `;
+      }
+    });
+  }
+
+  const transformPreprintSource = (source: string) => {
```

Comment:
Won't need this for ACL (lines 29-43), all the papers will be published at venues, not as preprints